components/python/salt/salt-PYVER.p5m

```diff
@@ -22,7 +22,10 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='Apache License 2.0'
 
-# Tornado does a lot of try/import/except, making pkgdepend unhapyy
+# Salt does a lot of try/import/except, making pkgdepend unhapyy
+# If one disables this line pkg will try to add dependencies to a giant list of dependencies that
```

Comment:
'will try to add dependencies to a giant list of dependencies' seems to be something non-english even for me
Perhaps, just remove second line and later comment?

Reply:
Yep that was me getting distracted.